en/simulation/README.md

```diff
@@ -87,10 +87,14 @@ The different parts of the system connect via UDP, and can be run on either the
 * PX4 uses a simulation-specific module to connect to the simulator's local TCP port 4560.
   Simulators then exchange information with PX4 using the [Simulator MAVLink API](#simulator-mavlink-api) described above.
   PX4 on SITL and the simulator can run on either the same computer or different computers on the same network.
+  :::note
```

Comment:
@TSC21 This is an update to Simulator docs that says a Simulator can use RTPS/DDS to interact with PX4 directly rather than via the simulator API. Can you confirm the note is accurate/useful?

Obviously it would be great to be able to have a topic "How connect a simulator using RTPS which identifies the topics etc). I figure we don't have time for that?